processing/src/main/java/io/druid/query/BaseQuery.java

```diff
@@ -145,10 +145,12 @@ public boolean getContextBoolean(String key, boolean defaultValue)
     return QueryContexts.parseBoolean(this, key, defaultValue);
   }
 
-  protected Map<String, Object> computeOverridenContext(Map<String, Object> overrides)
+  protected static Map<String, Object> computeOverriddenContext(
```

Comment:
@drcrallen asked me to keep compatibility of BaseQuery in earlier review of this PR: https://github.com/druid-io/druid/pull/4131#discussion_r108780846

I found this incompatibility because our extension broke :)

For annotation, there is an issue: #4044

Ok, I'll make a PR that fixes incompatibility